<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiskConfigurationDialog</class>
 <widget class="QDialog" name="DiskConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Disk configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0">
    <widget class="QLabel" name="meteringValveLabel">
     <property name="text">
      <string>Metering valve:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="insertValveLabel">
     <property name="text">
      <string>Insert valve:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QDoubleSpinBox" name="insertValveAngleBox">
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QDoubleSpinBox" name="hyperbaricValveAngleBox">
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="hyperbaricValveLabel">
     <property name="text">
      <string>Hyperbaric valve:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="angleLabel">
     <property name="text">
      <string>Angle</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="heater65LocationBox">
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QDoubleSpinBox" name="meteringValveAngleBox">
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="cuvetteOffsetBox">
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QDoubleSpinBox" name="hyperbaricValveRadiusBox">
     <property name="suffix">
      <string> mm</string>
     </property>
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="cuvetteSeparationLabel">
     <property name="text">
      <string>Cuvette separation:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="heater65LocationLabel">
     <property name="text">
      <string>Heater 65 location:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="heater95LocationLabel">
     <property name="text">
      <string>Heater 95 location:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="cuvetteSeparationBox">
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="heater95LocationBox">
     <property name="suffix">
      <string>°</string>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QDoubleSpinBox" name="meteringValveRadiusBox">
     <property name="suffix">
      <string> mm</string>
     </property>
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="cuvetteOffsetLabel">
     <property name="text">
      <string>Cuvette offset:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QDoubleSpinBox" name="insertValveRadiusBox">
     <property name="suffix">
      <string> mm</string>
     </property>
     <property name="minimum">
      <double>-99.989999999999995</double>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="radiusLabel">
     <property name="text">
      <string>Radius</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="currentLabel">
     <property name="text">
      <string>Current</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QDoubleSpinBox" name="hyperbaricValveCurrentBox">
     <property name="suffix">
      <string> A</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QDoubleSpinBox" name="meteringValveCurrentBox">
     <property name="suffix">
      <string> A</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QDoubleSpinBox" name="insertValveCurrentBox">
     <property name="suffix">
      <string> A</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="9" column="4">
    <widget class="QDoubleSpinBox" name="insertValveDurationBox">
     <property name="suffix">
      <string> s</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>30.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QDoubleSpinBox" name="meteringValveDurationBox">
     <property name="suffix">
      <string> s</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>30.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QDoubleSpinBox" name="hyperbaricValveDurationBox">
     <property name="suffix">
      <string> s</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="maximum">
      <double>30.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLabel" name="durationLabel">
     <property name="text">
      <string>Duration</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cuvetteOffsetBox</tabstop>
  <tabstop>cuvetteSeparationBox</tabstop>
  <tabstop>heater65LocationBox</tabstop>
  <tabstop>heater95LocationBox</tabstop>
  <tabstop>hyperbaricValveAngleBox</tabstop>
  <tabstop>hyperbaricValveRadiusBox</tabstop>
  <tabstop>hyperbaricValveCurrentBox</tabstop>
  <tabstop>hyperbaricValveDurationBox</tabstop>
  <tabstop>meteringValveAngleBox</tabstop>
  <tabstop>meteringValveRadiusBox</tabstop>
  <tabstop>meteringValveCurrentBox</tabstop>
  <tabstop>meteringValveDurationBox</tabstop>
  <tabstop>insertValveAngleBox</tabstop>
  <tabstop>insertValveRadiusBox</tabstop>
  <tabstop>insertValveCurrentBox</tabstop>
  <tabstop>insertValveDurationBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DiskConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DiskConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
